Riot/SupportingFiles/Riot.entitlements

```diff
@@ -36,7 +37,7 @@
 	<string>$(TeamIdentifierPrefix)$(CFBundleIdentifier)</string>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>$(APPLICATION_GROUP_IDENTIFIER)</string>
+		<string>group.au.gov.act.aether</string>
```

Comment:
Why can't this stay as the $(APPLICATION_GROUP_IDENTIFIER) variable?

Reply:
Apologies, I think this was an Xcode "helpful" change when trying to get provisioning profiles automatically generated to build to a device.  This has been reverted.